core/client/db/src/light.rs

```diff
@@ -757,125 +753,6 @@ pub(crate) mod tests {
 		}
 	}
 
-	#[test]
-	fn authorites_are_cached() {
```

Comment:
I feel bad about removing tests, but if cache isn't needed anymore I suppose it is OK.

Reply:
@svyatonik has reintroduced and activated the authorities cache again in #1669